apps/appsets/components.yaml

```diff
@@ -188,6 +188,27 @@ spec:
                     - repoURL: '{{index .metadata.annotations "uc_repo_git_url"}}'
                       targetRevision: '{{index .metadata.annotations "uc_repo_ref"}}'
                       path: 'components/chrony'
+                - component: openstack-exporter
+                  componentNamespace: monitoring
+                  skipComponent: '{{has "openstack-exporter" ((default "[]" (index .metadata.annotations "uc_skip_components") | fromJson))}}'
+                  sources:
+                    - repoURL: '{{index .metadata.annotations "uc_repo_git_url"}}'
+                      targetRevision: '{{index .metadata.annotations "uc_repo_ref"}}'
+                      path: 'components/openstack-exporter'
+                      ref: understack
```

Comment:
Since we're not using anything from here just drop this section.

Reply:
We'll need it in the future, but I've removed it for now.